notebooks/DrizzlePac/align_mosaics/align_mosaics.ipynb

```diff
@@ -0,0 +1,940 @@
+    "\n",
+    "Of the two IR filters, F160W has the largest number of point sources and therefore makes a good choice for the reference image. To generate source lists for matching, the TweakReg parameter `conv_width` should be set to approximately twice the FWHM of the PSF, ~2.5 pixels for IR observations and ~3.5 pixels for UVIS observations. TweakReg will automatically compute the standard deviation of the sky background (`skysigma`), so the number of sources in each catalog may be controlled simply by changing the ‘threshold’ parameter. \n",
+    "\n",
+    "In this example, TweakReg is run in ‘non-interactive’ mode (`interactive='False'`) so that the astrometric fit residuals and vectors diagrams will be saved as png files in the user’s local directory for inspection. Once the parameters have been fine-tuned and the fit looks adequate, users may run TweakReg a second time (see below) to update the image header WCS keywords by setting the parameter `updatehdr` to True."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tweakreg.TweakReg('*flt.fits',\n",
+    "                  imagefindcfg={'threshold':50,'conv_width':2.5},\n",
+    "                  expand_refcat=True, enforce_user_order=False,\n",
+    "                  shiftfile=True, outshifts='shift160_flt.txt',\n",
+    "                  searchrad=2.0, ylimit=0.3,\n",
+    "                  updatehdr=False, interactive=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4b. Inspect the shift file to verify the pointing residuals \n",
+    "\n",
+    "With the `threshold` parameter set to 50 sigma, TweakReg finds ~500 objects per FLC image, matches the individual catalogs, and computes residual shifts between exposures. These offsets (given in pixels at the native IR scale=0.1283”/pixel) are recorded in an output “shift file” which is shown below.  Note that the fourth row of the shift file corresponds to the reference image ‘ick902neq_flt.fits’ which was automatically selected by TweakReg. The computed offsets reflect updates to the header WCS required to correct for small pointing errors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "T=Table.read('shift160_flt.txt', format='ascii.no_header', names=['file','dx','dy','rot','scale','xrms','yrms'])\n",
```

Comment:
```suggestion
    "T = Table.read('shift160_flt.txt', format='ascii.no_header', names=['file','dx','dy','rot','scale','xrms','yrms'])\n",
```

Also, I strongly recommend *not* using the name `T` - that's both confusing because it's a capital letter, and it's a single character, both recommended against by PEP8.  Perhaps `shift_table`?

Reply:
Funny, I independently just changed it to `shift_table` in #34 before I even saw this.